src/utils/validation.ts

```diff
@@ -124,6 +124,7 @@ function validateElement(
 const emailPatternStr =
   "^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]+)+$";
 const emailPattern = new RegExp(emailPatternStr);
+// TODO: deprecate and support international format
 const phonePattern = /^\d{10}$/;
```

Comment:
Can we just get rid of this now and replace with the validation method below.  It seems like firebase and the is_type comparison would behave roughly the same using it instead?

Reply:
think @obmelvin discussed this with you offline, we'll do the login refactor in a separate change